utils/spacewalk-hostname-rename

```diff
@@ -390,6 +426,25 @@ function re-generate_server_ssl_certificate {
     fi
 }
 
+function refresh_pillar {
+    echo -n "Refreshing Salt minion pillar data, may take a while ... " | tee -a $LOG
+    for ID in `spacecmd -q api system.listSystemsWithEntitlement -- -A "salt_entitled" -F "%(id)s"`
+    do
+        if [ -n "$SIDS" ]
+        then
+            IDS="$SIDS,$ID"
+        else
+            IDS="$SID"
+        fi
+    done
+    SKIPPED=`spacecmd -q api system.refreshPillar -- -A "[\\\\\"general\\\\\",[$SIDS]]"`
```

Comment:
Where does `$SIDS` come from?
I see only `ID` and `IDS` variable been set.

Reply:
`$IDS` is empty (undefined to be more precise) at the beginning of the loop. This helps handling properly the comma separation between values

Reply:
oops... looks like an incomplete rename to make `IDS` more understandable